VAMobile/src/components/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react'
+import { Linking, Platform, Pressable, ViewStyle } from 'react-native'
+
+import { BackgroundVariant, Box, BoxProps, TextView, VAIcon } from 'components'
+import { useTheme } from 'utils/hooks'
+
+import colors from '../styles/themes/VAColors'
+
+interface AnnouncementBannerProps {
+  /** Text for announcement title */
+  title: string
+  /** link to navigate to, excluding the prefix */
+  link: string
+}
+
+const AnnouncementBanner: FC<AnnouncementBannerProps> = ({ title, link }: AnnouncementBannerProps) => {
+  const theme = useTheme()
+
+  const boxProps: BoxProps = {
+    py: theme.dimensions.formMarginBetween,
+    px: theme.dimensions.cardPadding,
+    backgroundColor: theme.colors.buttonBackground.announcementBanner as BackgroundVariant,
+    style: {
+      shadowColor: colors.black,
+      ...Platform.select({
+        ios: {
+          shadowOffset: { width: 0, height: 6 },
+          shadowOpacity: 0.2,
+          shadowRadius: 8,
+        },
+        android: {
+          elevation: 8,
+        },
+      }),
+    },
+  }
+
+  const pressableStyles: ViewStyle = {
+    flexDirection: 'row',
+  }
+
+  return (
+    <Box {...boxProps}>
+      <Pressable
+        style={pressableStyles}
+        onPress={() => Linking.openURL(`https://${link}`)}
```

Comment:
Since we should be storing and referencing links from the `constant.env` file, we probably don't need to prefix the link since the consts will already be the full URL